CRVResponse/fcl/prolog_extracted_v01.fcl

```diff
@@ -0,0 +1,16 @@
+#include "CRVResponse/fcl/prolog.fcl"
+
+physics.producers.CrvPhotons.crvStepModuleLabels  : ["CrvSteps"]
+physics.producers.CrvPhotons.crvStepProcessNames  : ["CrvStepsConverter"]
+physics.producers.CrvPhotons.CRVSectors           : ["EX","T1","T2"]
```

Comment:
This naming convention is a bit confusing.
We already have modules T1, T2 and E in:
Mu2eG4/geom/crv_counters_v07.txt
You might want to consider using T, D, L.
I don't have a strong prefernce though...

Reply:
I'm fine with any suggested names, I started with EX for extracted, but this was just a starting point.
Would you suggest `EX --> T`, `T1 --> D`, and `T2 --> L`?

Reply:
Is there a place ( in code .hh .cc or on the wiki ) that defines what all of the abbreviations mean?  If so please point me to it.  If not please make one.  If you decide to put it on the wiki, reference the wiki page in the code.    Do you throw an exception if someone enters a name that you don't recognize?

Reply:
I agree that the documentation or pointers thereto would be good to have.

Reply:
Yuri: since you approved this PR, I presume you are OK with the names as is and will merge.